package.json

```diff
@@ -57,20 +57,19 @@
 		"@babel/preset-react": "^7.13.13",
 		"@babel/preset-typescript": "^7.13.0",
 		"@babel/runtime": "^7.14.0",
-		"@storybook/addon-actions": "^6.4.8",
-		"@storybook/addon-docs": "^6.4.8",
-		"@storybook/addon-essentials": "^6.4.8",
-		"@storybook/addon-links": "^6.4.8",
-		"@storybook/react": "^6.4.8",
-		"@testing-library/jest-dom": "^5.11.10",
-		"@testing-library/react": "^11.2.6",
-		"@testing-library/react-hooks": "^7.0.0",
-		"@testing-library/user-event": "^13.1.9",
-		"@types/jest": "^26.0.23",
-		"@types/material-ui": "^0.21.8",
-		"@types/react-dom": "^17.0.5",
-		"@types/react-select": "^5.0.1",
-		"@types/styled-components": "^5.1.9",
+		"@storybook/addon-actions": "^6.5.9",
+		"@storybook/addon-docs": "^6.5.9",
+		"@storybook/addon-essentials": "^6.5.9",
+		"@storybook/addon-links": "^6.5.9",
```

Comment:
De adon @storybook/addon-a11y en @storybook/addon-interactions zijn dingen waar ik graag meer mee wil doen, maar dat is voor later misschien. Vooral accessibility is vrij laagdrempelig. Of heb je een reden om die niet te gebruiken?

Reply:
I'm totally happy to introduce these. But that's out of scope for this PR IMO